databricks_cli/clusters/cli.py

```diff
@@ -331,6 +331,44 @@ def cluster_events_cli(api_client, cluster_id, start_time, end_time, order, even
         click.echo(tabulate(_cluster_events_to_table(events_json), tablefmt='plain'))
 
 
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--cluster-id', cls=OneOfOption, one_of=CLUSTER_OPTIONS,
+              type=ClusterIdClickType(), default=None, help=ClusterIdClickType.help)
+@click.option('--cluster-name', cls=OneOfOption, one_of=CLUSTER_OPTIONS,
+              type=ClusterIdClickType(), default=None, help=ClusterIdClickType.help)
+@click.option('--local-port', type=click.INT, help="the local port")
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def tunnel_cli(api_client, cluster_id, cluster_name, local_port):
+    """
+    Start a secure TCP tunnel to a cluster over Databricks' identity proxy.
+    """
+    if sys.version_info < (3, 6):
+        raise RuntimeError("The tunneling command is not supported on Python version < 3.6")
+    if not api_client.config:
+        raise RuntimeError("Unable to initialize client."
+                           "Please check if the credentials are configured properly.")
```

Comment:
oh yeah, this isn't needed, but since we require pat (require to create tunnel session), I'll modify api_client to store this value and then check it in the tunneling cli.